C#Backend/InpatientTherapySchedulingProgram/Controllers/HoursWorkedController.cs

```diff
@@ -79,45 +83,39 @@ public async Task<IActionResult> PutHoursWorked(int id, HoursWorked hoursWorked)
         [HttpPost]
         public async Task<ActionResult<HoursWorked>> PostHoursWorked(HoursWorked hoursWorked)
         {
-            _context.HoursWorked.Add(hoursWorked);
             try
             {
-                await _context.SaveChangesAsync();
+                await _service.AddHoursWorked(hoursWorked);
             }
             catch (DbUpdateException)
             {
-                if (HoursWorkedExists(hoursWorked.HoursWorkedId))
-                {
-                    return Conflict();
-                }
-                else
-                {
-                    throw;
-                }
+                throw;
             }
 
             return CreatedAtAction("GetHoursWorked", new { id = hoursWorked.HoursWorkedId }, hoursWorked);
         }
 
         // DELETE: api/HoursWorkeds/5
         [HttpDelete("{id}")]
-        public async Task<ActionResult<HoursWorked>> DeleteHoursWorked(DateTime id)
+        public async Task<ActionResult<HoursWorked>> DeleteHoursWorked(int id)
```

Comment:
Try to be consistent here and use hoursWorkedId for all instances of the word id.